hyper/client/aws/ec2.go

```diff
@@ -526,12 +534,11 @@ func GetInstanceForStudy(studyName string, remoteCfg hyperdriveTypes.EC2ComputeR
 func IsStructureEmpty(i types.Instance) bool {
 	return reflect.DeepEqual(i, types.Instance{})
 }
-func StartJupyterEC2(manifestPath string, remoteCfg hyperdriveTypes.EC2ComputeRemoteConfiguration, ec2Type string, amiID string, jupyterLaunchOptions hyperdriveTypes.JupyterLaunchOptions, syncOptions hyperdriveTypes.WorkspaceSyncOptions) {
-	startupScript := getEc2StartScript(version, jupyterLaunchOptions, syncOptions, remoteCfg)
-	StartServer(manifestPath, remoteCfg, ec2Type, amiID, startupScript, jupyterLaunchOptions.HostPort)
+func StartJupyterEC2(manifestPath string, remoteCfg hyperdriveTypes.EC2ComputeRemoteConfiguration, ec2Type string, amiID string, jupyterLaunchOptions hyperdriveTypes.JupyterLaunchOptions, syncOptions hyperdriveTypes.WorkspaceSyncOptions, serverType EC2Type) {
```

Comment:
I think instead of adding another parameter to StartJupyterEC2, we add a StartHyperpackageEc2 or something similar. Both will call StartServer, but pass in a different start script (one can call a GetJupyterEC2StartScript and the other GetHyperpackageEC2StartScript). Then we can move the log message conditional out from StartServer  up in to each of the specialized StartServer functions